Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -843,31 +1063,79 @@ eval RemoveChat state = do
     pure $ NoAction
   ]
 
+eval WaitingInfo state = continue state{data{waitTimeInfo  = true }}
+
+eval (RequestInfoCardAction RequestInfoCard.Close) state = continue state { data  {waitTimeInfo =false}}
+
+eval (SendQuickMessage chatSuggestion) state = do
+  if state.props.canSendSuggestion then do
+    _ <- pure $ sendMessage chatSuggestion
+    let _ = unsafePerformEffect $ logEvent state.data.logField $ "ny_" <> STR.toLower (STR.replaceAll (STR.Pattern "'") (STR.Replacement "") (STR.replaceAll (STR.Pattern ",") (STR.Replacement "") (STR.replaceAll (STR.Pattern " ") (STR.Replacement "_") chatSuggestion)))
```

Comment:
can't we log it as it is?

Reply:
The logs contains `,` `?` etc.., cannot log those to firebase_logs